core/Macros.carp

```diff
@@ -119,6 +119,103 @@
   (defndynamic collect-into [xs f]
     (list 'quote
       (collect-into-internal xs (f) f)))
+
+  (doc empty?
+    "Returns true if the provided data literal is empty, false otherwise.")
+  (defndynamic empty? [xs]
+    (if (= 0 (length xs))
```

Comment:
Can’t we just remove the `if` and return the expression directly?

Reply:
done! ba0bfe7